app/models/service_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ServiceUpdate
+  CLAIMS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/claims_service_updates.yml").freeze
+  PLACEMENTS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/placements_service_updates.yml").freeze
+
+  include ActiveModel::Model
+
+  attr_accessor :date, :title, :content
```

Comment:
But if you want to use it you probably want to remove attr_accessor.
```suggestion
class ServiceUpdate
  include ActiveModel::Attributes

  attribute :date, :string
  ...
end
```